modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/ValidateClientExtensionTask.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.gradle.plugins.workspace.task;
+
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonParser;
+
+import com.liferay.gradle.plugins.workspace.internal.client.extension.ClientExtension;
+import com.liferay.gradle.plugins.workspace.internal.util.StringUtil;
+import com.liferay.gradle.util.ArrayUtil;
+import com.liferay.petra.string.StringPool;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+import java.util.Objects;
+
+import org.gradle.api.GradleException;
+import org.gradle.api.Project;
+import org.gradle.api.tasks.TaskAction;
+
+/**
+ * @author Anderson Luiz
+ */
+public class ValidateClientExtensionTask {
+
+	@TaskAction
+	public void validate(ClientExtension clientExtension, Project project) {
```

Comment:
I'm not sure, but if you're calling the `validate` method directly from `ClientExtensionProjectConfigurator`, this `@TaskAction` means nothing.

Is it useful, @drewbrokke ?